tools/mob_suite/mob_recon.xml

```diff
@@ -12,78 +13,133 @@
     #set $named_input = re.sub(r'(\s|\(|\)|:|!)', '_', str($input.element_identifier)+'.fasta')
     ln -s '$input' '$named_input' &&
     
-    mob_recon  --num_threads \${GALAXY_SLOTS:-4}  --infile '${named_input}'
+
+    mob_recon  --num_threads \${GALAXY_SLOTS:-4}  --infile '${named_input}' --run_typer
+
     #if str($adv_param.unicycler_contigs) == "True":
       --unicycler_contigs 
     #end if   
-    #if str($adv_param.run_circlator) == "True":
-      --run_circlator 
-    #end if 
-    #if str($adv_param.min_length_condition.min_length_param) == "True":
-      --min_length '${adv_param.min_length_condition.min_length_value}'
+
+    #if str($adv_param.run_overhang) == "True":
+      --run_overhang
     #end if 
-    --run_typer --min_rep_evalue '${adv_param.min_rep_evalue}'
+    
+    #if $adv_param.debug:
+      --debug
+    #end if
+
+    #if $adv_param.plasmid_db
+   --plasmid_db $adv_param.plasmid_db
```

Comment:
Is `$adv_param.plasmid_db` a free form field? If so, surround with `'`

Reply:
Could you provide me reference to the free form fields in Cheetah language? Thank you

Reply:
Sorry, what I meant is that is the user able to input a file for `plasmid_db`? If so, then there should be `'` that surrounds the variable as the user could choose a weird file name that may escape your command.

Reply:
Also I see that you added quotes, resolving this comment.

Reply:
Ah, I see. The literal value is safeguard. Indeed some file names could have  weird symbols like`' or &&`, etc.